assets/src/edit-story/app/media/utils/getResourceSize.js

```diff
@@ -27,16 +32,23 @@
  * @return {Object} The resource's size (width and height).
  */
 function getResourceSize(
-  width,
-  height,
-  posterGenerated,
-  posterWidth,
-  posterHeight
+  width = 0,
+  height = 0,
+  posterGenerated = false,
+  posterWidth = 0,
+  posterHeight = 0
 ) {
   if (posterGenerated && posterWidth && posterHeight) {
     return { width: posterWidth, height: posterHeight };
   }
-  return { width, height };
+  if (width || height) {
+    return { width, height };
+  }
+
+  return {
```

Comment:
There might be other ways of doing this, but defaulting to these values everywhere makes a lot of sense to me. There are lots of places where height and width of image / width could be missing. Default makes sense to me.

Reply:
We might want to consider adding tests for the util functions like this and `getResourceFrom*`, just so the expected behavior is documented through tests.

Reply:
I have added some unit tests for `getResourceSize`. It isn't used in to many places, so these should cover it.